apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx

```diff
@@ -38,9 +38,10 @@ export const CreateExchangeOrderScreen = () => {
   const {orderType, canExchange, providerId, provider, amount, referralLink: managerReferralLink} = useExchange()
 
   const providers = useExchangeProvidersByOrderType({orderType, providerListByOrderType: provider.list.byOrderType})
-  const providerSelected = Object.fromEntries(providers)[providerId]
-  const fee = providerSelected.supportedOrders?.[orderType]?.fee ?? 0
-  const Logo = providerSelected.id === 'banxa' ? BanxaLogo : EncryptusLogo
+  const providerSelected = new Map(providers).get(providerId)
```

Comment:
eslint was getting crazy, (current version is not compatible with typescript) only to suffice eslint check